checkmate/routes.py

```diff
@@ -6,7 +6,7 @@ def add_routes(config):
 
     config.add_route("root", "/")
 
-    config.add_route("get_status", "/_status")
+    config.add_route("status", "/_status")
```

Comment:
Renaming this to just `"status"` to match the path and the name used for this route in our other apps.

`get_status` isn't really an appropriate name for a route unless you're going to add the `request_method` param to the route config, since without `request_method` views for all HTTP methods (`GET`, `POST`, ...) could be attached to this route.